engine/gbc_only.asm

```diff
@@ -13,7 +13,7 @@ GBCOnlyScreen: ; 4ea82
 	ld de, wd000
 	ld a, [rSVBK]
 	push af
-	ld a, 0
+	ld a, 0 ; this is tantamount to selecting Bank 1
```

Comment:
Please clarify that this isn't a typo; writing 0 to rSVBK has the same effect as 1. ([SVBK](http://gbdev.gg8.se/files/docs/mirrors/pandocs.html#videodisplay))